docs/running-your-own-experiment/running_an_experiment.md

```diff
@@ -63,10 +63,10 @@ cloud_project: $PROJECT_NAME
 cloud_compute_zone: $PROJECT_ZONE
 
 # The Google Cloud Storage bucket that will store most of the experiment data.
-cloud_experiment_bucket: gs://$DATA_BUCKET_NAME
+experiment_filestore: gs://$DATA_BUCKET_NAME
 
-# The bucket where HTML reports and summary data will be stored.
-cloud_web_bucket: gs://$REPORT_BUCKET_NAME
+# The filestore where HTML reports and summary data will be stored.
```

Comment:
Don't like this change, please change it to `Google Cloud Storage bucket`. It makes sense to talk about GCS here since this document is about running experiments in cloud.